tests/rptest/tests/scaling_up_test.py

```diff
@@ -563,22 +574,22 @@ def test_moves_with_local_retention(self, use_topic_property):
         def print_disk_usage(usage):
             for n, b in usage.items():
                 self.logger.info(
-                    f"node: {n} total partitions size: {b/(1024*1024)} Mb")
+                    f"node: {n} total partitions size: {b/(1024*1024):02} Mb")
```

Comment:
nit and out of scope: Mb is megabit, MB is megabyte (1000^2), MiB is Mebibyte (1024^2)